application-templates/javascript/assets/vite.config.js

```diff
@@ -0,0 +1,23 @@
+import { resolve, dirname } from 'path';
+import { fileURLToPath } from 'url';
+import { defineConfig } from 'vite';
+import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js';
```

Comment:
Do you know if we need to include this? I would prefer to be unopinionated as much as possible